statefun-flink/pom.xml

```diff
@@ -56,13 +56,68 @@ under the License.
             <id>apache.snapshots</id>
             <name>Apache Development Snapshot Repository</name>
             <url>https://repository.apache.org/content/repositories/snapshots/</url>
-            <releases><enabled>false</enabled></releases>
-            <snapshots><enabled>true</enabled></snapshots>
+            <releases>
+                <enabled>false</enabled>
+            </releases>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
         </repository>
     </repositories>
 
     <dependencyManagement>
+        <!-- Flink -->
         <dependencies>
+            <dependency>
+                <groupId>org.apache.flink</groupId>
+                <artifactId>flink-streaming-java_2.11</artifactId>
```

Comment:
I would heavily recommend also defining the version in here; DependencyManagement entries without versions are not really intended and can break in weird ways when plugins try to resolve dependencies.
Depends a bit of course on which plugins you want to use in the future; we specifically ran into this when using the `japicmp` plugin which verifies API compatibility; which I suppose we may add in the future(?).

Reply:
Thanks! your recommendation make sense, I will address that.